CMakeLists.txt

```diff
@@ -894,30 +894,19 @@ endif()
 
 set(SPIRV OFF)
 
-if(ENABLE_LLVM)
-  option(ENABLE_SPIR "Enable SPIR support (default ON when available)" ON)
+# drivers that consume SPIRV directly don't need LLVM or LLVM-SPIRV;
+# those that consume SPIRV indirectly need both, but it might be useful to disable LLVM-SPIRV translator
```

Comment:
Reminds me that we should now drop the old LLVM versions since it's quite early in the release cycle. Can you create a single commit that does it and cleans up some LLVM version macro checks?

Reply:
sure